app/controllers/admin/questions_controller.rb

```diff
@@ -58,4 +58,4 @@ def check_admin
     redirect_to authenticated_root_path, notice: 'Access Denied'
   end
 
-end
+end
```

Comment:
changes like this should not be committed
- first of all file ending isn't correct
- if file does not have changed lines do not add it to commit